functions/changeDeleteFlag.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+function changeDeleteFlag(int $id, PDO $db) {
```

Comment:
Doc Block!

Reply:
Because this funtion doesn't return anything, you can return type hint it as a `void`